files/monit_hwraid

```diff
@@ -21,7 +21,7 @@ type -ap $raid >& /dev/null || {
     exit 1
 }
 
-stat=$($raid $args 2> /dev/null | grep $patt)
+stat=$($raid "$args" 2> /dev/null | grep $patt)
```

Comment:
That change is not right, we want $args to be split on whitespace.
(In general I find shellcheck mainly just an exercise in pedantry.)

Reply:
Copy. I've fixed.  I often find shellcheck useful and it at least imposes something of a consistent style across repos.